src/config/dynamic-theme-fixes.config

```
@@ -9154,6 +9154,39 @@ CSS
 
 ================================
 
+my.contabo.com
+
+CSS
+#mainmenu {
+    background-color: rgb(0, 0, 111) !important;
+    border-radius: 20px 0px 0px 20px !important;
+    box-shadow: -2px -2px, -2px 2px, 0px 2px, 0px -2px !important;
+}
+#mainmenu > div {
+    background-image: none !important;
+}
+#submenu {
+    background-color: #977122 !important;
```

Can this be implemented along the lines of:
```suggestion
    background-color: ${gray} !important;
```